requirements/edx/base.txt

```diff
@@ -90,6 +91,7 @@ django-pyfs==2.0
 django-ratelimit-backend==1.1.1
 django-ratelimit==1.1.0
 django-require==1.0.11
+django-rest-framework==0.1.0
```

Comment:
@jcdyer just wondering, why is this now needed here?

Reply:
It shouldn't be.  DRF used to be called this, and changed their PyPI name to djangorestframework.  edx-platform uses the dashless name, and the completion aggregator used the old name (but has been fixed now), so there's a conflict.  I'm working on resolving this.